docs/protocol/warp-routes/deploy-multicollateral-warp-routes.mdx

```diff
@@ -0,0 +1,117 @@
+# Deploy Multi-Collateral Warp Routes
+
+This guide walks you through how to use Hyperlane Warp Routes to create multi-collateral token bridges, allowing assets to be sourced from multiple chains instead of a single origin.
+
+## Prerequisites
+
+To complete this walkthrough, you should have the following:
+
+1. Two collateral chains and one synthetic chain of your choice, between which you’d like to deploy the multi-collateral warp route. Make sure to have some funds on the chains to cover deployment gas fees.
+2. The token(s) to be used as collateral.
+3. An installed instance of the [Hyperlane CLI](https://docs.hyperlane.xyz/docs/reference/cli) and a wallet private key set as the `HYP_KEY` env var funded on your origin and destination networks.
+
+## Walkthrough
+
+### Primer: Multi-Collateral Warp Routes
+
+Multi-Collateral Warp Routes are an extension of Hyperlane's Warp Routes, allowing liquidity to be sourced from multiple chains instead of a single source.
```

Comment:
i wouldn't say it's an extension. extensions, imo, are warp routes like xerc20, fiattoken. MCWR are more of a configuration with 2+ collateral types